vocabularies/UI.xml

```diff
@@ -1609,6 +1608,17 @@ This is in contrast to [DataFieldForIntentBasedNavigation](#DataFieldForIntentBa
         <Annotation Term="Core.Description" String="The contents of this property must not be propagated to the app-to-app navigation context" />
       </Term>
 
+      <Term Name="Timezone" Type="Edm.String" AppliesTo="EntityContainer Property Parameter">

```

Comment:
How is this supposed to work when applied to the entity container? Static user-independent values don't make much sense, and user-dependent values would require an absolute path to a property of a "user-settings" singleton.

Remove this for now and just apply it to properties and parameters?

Reply:
Can't the value be user-dependent (just like the texts in the metadata depend on the user's language)? It could be the timezone from the user settings.

If not, then let's apply it only to properties and parameters.

Reply:
Neither RAP nor CAP support user-dependent $metadata.